docs/examples/marketplace.md

```diff
@@ -0,0 +1,267 @@
+---
+sidebar_label: NFT Marketplace
+sidebar_position: 10
+---
+
+# NFT Marketplace
+
+## Introduction
+NFT marketplace is a contract where you can buy and sell non-fungible tokens for fungible tokens or Gear native value. The contract also supports holding the NFT auctions and making/accepting purchase offers on NFTs.
```

Comment:
What about marketplace application? I suggest to add it in this article once video will be released.